src/pkg/blob/models/blob.go

```diff
@@ -83,6 +85,11 @@ func (b *Blob) IsForeignLayer() bool {
 	return b.ContentType == schema2.MediaTypeForeignLayer
 }
 
+// IsManifest returns true if the blob is manifest layer
+func (b *Blob) IsManifest() bool {
```

Comment:
Also need to take into account the manifest of oci images?

Reply:
added

Reply:
added.